molgenis_documentation/user_documentation.md

```diff
@@ -947,8 +1004,171 @@ To get the feel of how certain styles look, you can select it in the dropdown me
+
+###### <a name="upload-screens-result-steps"></a>Steps
+
+1. [Optional] Select the view permission for the root_hospital_cities entity.
+2. [Optional] Click on the save button.
+3. Click on the finish button.
+
+![Upload file screen](images/plugin-upload/result-screen.png?raw=true, "Result")
+
+##### <a name="dataexplorer-reports"></a>Data-explorer "Reports"
+The reports functionality is made for overriding the default instance view or to add a instances tab in the Data-explorer. Overriding the views or adding a tab is possible by creating a new FreemarkerTemplate entity with the right name convention. In this short tutorial I will show you how to achieve this.
+
+There are two ways to create your own reports: 
+
+1. Overriding the default instance view.
+2. Add a instances view tabs.
+
+You will need:
+
+1. A data set: "Advanced data example" ([download](data/advanced_data_example_v20151104.xlsx)). Upload this dataset into your MOLGENIS instance see the [Upload](#upload) section.
+
+###### Override the entity view
+
+Steps:
+
+1. Go to the data explorer
+2. Select the "cities" entity via the entity select dropdown.
+3. The entity view modal is opened when you click on the ![View entity report button](images/reports/view-entityreport-button.png?raw=true, "Entity view") button".
+4. The default view will be: ![View entity report default](images/reports/default-entityreport-view.png?raw=true, "Entity view")
+5. Lets upload our own template. 
+	a. Go to the data explorer and select the FreemarkerTemplate entity.
+	b. Click on the ![add](images/add.png?raw=true, "add") button. In the modal you fill in:
+		* Name: view-entityreport-specific-root_hospital_cities.ftl (view-entityreport-specific-\<Full entity name>.ftl)
+		* Value: "\<div>City name: ${entity.get('cityName')}\</div>"
+	![view-Cities-entitiesreport](images/reports/view-entityreport-specific-root_hospital_cities.png?raw=true, "view-Cities-entitiesreport")
+6. Repeat steps 2 and 3. 
+7. The new view will be: ![View entity report custom](images/reports/custom-entityreport-view.png?raw=true, "Entity view")
+
+###### Add a instances view tab
+1. Go to the data explorer select the "cities" entity via the th entity select dropdown.
+2. Lets upload our own template. 
+	a. Go to the data explorer and select the FreemarkerTemplate entity.
+	b. Click on the ![add](images/add.png?raw=true, "add") button. In the modal you fill in:
+		* Name: view-Cities-entitiesreport.ftl (view-\<template name>-entitiesreport.ftl)
+		* Value: "\<div>My cities\</div>"
+		![view-Cities-entitiesreport](images/reports/view-Cities-entitiesreport.png?raw=true, "view-Cities-entitiesreport")
+3. Click on the settings icon ![Settings](images/settings.png?raw=true, "Settings")
+	a. Check: Modules -> Data -> Reports -> Yes
+	b. Set: Reports -> Reports -> root_hospital_cities:Cities
+		* root_hospital_cities is the entity name.
+		* Cities is the template name.
```

Comment:
Got it working, entity report looks like this:
```
<link rel="stylesheet"
      href="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.css"/>

<div id="map" style="width: 600px; height: 400px"></div>

<script>
    function showMap() {
        var map = L.map('map').setView([38, -80], 4);

        L.tileLayer('https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token=pk.eyJ1IjoibWFwYm94IiwiYSI6IjZjNmRjNzk3ZmE2MTcwOTEwMGY0MzU3YjUzOWFmNWZhIn0.Y8bhBaUMqFiPrDRW9hieoQ', {
            maxZoom: 18,
            attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, ' +
            '<a href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, ' +
            'Imagery © <a href="http://mapbox.com">Mapbox</a>',
            id: 'mapbox.streets'
        }).addTo(map);

    <#list datasetRepository.iterator() as city>
        L.marker([${city.get("lat")},${city.get("lng")}]).addTo(map)
                .bindPopup("${city.get("cityName")}").openPopup();
    </#list>
    }
    $.getScript("http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.js", showMap);
</script>
```

Need to add lat/lng attributes to the emx file for the cities entity.